### DISCUSSION TITLE
Anyone Working with Jacdac and Disabilities or Education ?

### DISCUSSION THREAD

**Discussion by FrankSchmidtDDS** [microsoft/jacdac#1350] [Q&A]
Hi All,
I have an invitation from our county disabilities board to present on our volunteer robotics/coding class. We have about 100 videos on Jacdac system on YouTube now. I wonder if anyone is working with Jacdac (or knows of someone) with persons with disabilities or in kids education ?  We have both Kits A and B and RP 2040 from Kittenbot. Steve Hodges at one time talked about this subject. Any contacts would be appreciated. I can give my email address also.

**Comment by thomasjball** [comment]
Frank, I do recall that (according to @sehodges), Makers Making Change was using Jacdac for prototyping assistive tech: https://www.makersmakingchange.com/

**Comment by FrankSchmidtDDS** [comment]
Thanks Tom,
 Please keep me in mind if you hear of anybody doing this; I'll respond to him if Steve Hodges contacts me with anything. Anyone can leave a comment on our YouTube channel to contact me: https://www.youtube.com/@JacdacMakersKidsEdPlugan-bt4ut
Best,
Frank

________________________________

**Comment by FrankSchmidtDDS** [comment]
Unfortunately , YouTube cancelled our YouTube channel for Jacdac so the above link was broken. I made a new channel and replaced the broken link above. Here's our Jacdac channel again. It would be nice to have a "turn off LEDs" block in MakeCode for the Jacdac RGB  LED strip like we have for the RGB led ring. https://www.youtube.com/@JacdacMakersKidsEdPlugan-bt4ut
